perf/istio/setup.sh

```diff
@@ -116,6 +123,9 @@ shift
 setup_admin_binding
 install_istio "${WD}/tmp" "${release}" $*
 install_gateways
+if [[ ! -z "${WORKLOAD_FILE}" ]]; then
+  inject_workload "${WORKLOAD_FILE}"
```

Comment:
Don’t install workloads here. 
We already have several workloads.
If it does not suit your needs create a sibling chart dir with your test. Looks at other peer directories of istio for example.

Reply:
Revised based on the comment.